tests/PageObjects/GamePage.ts

```diff
@@ -9,4 +9,24 @@ export class GamePagePO {
     await this.page.waitForTimeout(1500);
     await this.page.keyboard.press('Enter');
   }
+
+  public async restartSong() {
+    await this.page.getByTestId('button-restart-song').click();
+  }
+
+  public get playersScoreElement() {
+    return this.page.getByTestId('players-score');
+  }
+
+  public get currentPlayersScore() {
+    return this.playersScoreElement.getAttribute('data-score');
+  }
+
+  public async waitForPlayersScoreToBeGreaterThan() {
```

Comment:
It should expect an argument to specify greater than what value the score should be
```
waitForPlayersScoreToBeGreaterThan(expected: number)
```